_posts/2019-12-20-festive-message.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "A look back at 2019"
+date: 2019-12-20
+author: strimzi_developers
+---
+
+The last year has been a busy one for Strimzi with many highlights. 
+We wanted to reflect on and celebrate a few of them.
+
+<!--more-->
+
+Many features have been added during the year. Here are some of the key ones:
+
+* Storage - resizing persistent volumes and support for JBOD (including adding and removing volumes)
+* Monitoring - support for [Jaeger](https://www.jaegertracing.io/) tracing, [Prometheus](https://prometheus.io/) metrics and exposing consumer lag using [Kafka Exporter](https://github.com/danielqsj/kafka_exporter)
+* Support for OAuth2 authentication
+* HTTP access through the Kafka Bridge
+* New mechanisms for off-cluster access
+
+Next year, Strimzi will accelerate and we are looking forward to adding support for:
+
+* Cluster rebalancing using [Cruise Control](https://github.com/linkedin/cruise-control)
+* An Operator for handling Kafka Connect connectors as native Kubernetes resources
+* Enabling ZooKeeper rolling upgrade
+* Support for replication using MirrorMaker2
+
+We are also proud of the non-technical achievements Strimzi made during the year, especially being accepted into the [Cloud Native Computing Foundation](https://www.cncf.io/).
+As part of this we also developed the first governance rules for Strimzi’s growing community. 
+This saw Tom Bentley, Paolo Patierno and Jakub Scholz become the initial group of maintainers. 
+They work tirelessly to both review code and provide guidance and encouragement to new contributors. 
```

Comment:
Honestly, on some days I do get tired ;-)

Reply:
Only on some days? That means that you don't work enough on the other days! :-D